libs/api/domains/driving-license-book/src/drivingLicenseBook.service.ts

```diff
@@ -86,6 +94,9 @@ export class DrivingLicenseBookService {
   async getStudentsForTeacher(
     user: User,
   ): Promise<DrivingLicenseBookStudentForTeacher[]> {
+    this.logger.debug(
+      `driving-license-book: Getting student for teacher ${user}`,
+    )
```

Comment:
This might log as `"...for teacher [Object object]"`.

The logger supports objects like
```typescript
this.logger.debug({ message: 'getting student for teacher', user: user })
```